### PR TITLE
Update development opam files of coq-tactician

### PR DESCRIPTION
**Pull request by LasseBlaauwbroek**

### CODE REVIEW

File: extra-dev/packages/coq-tactician/coq-tactician.8.11.dev/opam [hunk 1]
```diff
@@ -1,43 +1,51 @@
...
+authors: ["Lasse Blaauwbroek <lasse@blaauwbroek.eu>"]
 homepage: "https://coq-tactician.github.io"
-dev-repo: "git+https://github.com/coq-tactician/coq-tactician"
 bug-reports: "https://github.com/coq-tactician/coq-tactician/issues"
-maintainer: "Lasse Blaauwbroek <lasse@blaauwbroek.eu>"
-authors: "Lasse Blaauwbroek <lasse@blaauwbroek.eu"
 depends: [
   "ocaml" {>= "4.08"}
-  "dune" {>= "2.5"}
+  "dune" {>= "2.5" & build}
```
Comment (palmskog) [inline extra-dev/packages/coq-tactician/coq-tactician.8.11.dev/opam:20]:
```suggestion
  "dune" {>= "2.5"}
```
Please do not use `build` for Dune. It won't work. See issues in the Dune repo for a long discussion of why.

File: extra-dev/packages/coq-tactician/coq-tactician.8.11.dev/opam [hunk 2]
```diff
@@ -1,43 +1,51 @@
 opam-version: "2.0"
-name: "coq-tactician"
-synopsis: "Tactician: A Seamless, Interactive Tactic Learner and Prover for Coq"
+version: "8.11.dev"
```
Comment (palmskog) [inline extra-dev/packages/coq-tactician/coq-tactician.8.11.dev/opam:2]:
```suggestion
```
Version is already defined by the file path.

File: extra-dev/packages/coq-tactician/coq-tactician.8.12.dev/opam [hunk 3]
```diff
@@ -1,43 +1,51 @@
 opam-version: "2.0"
-name: "coq-tactician"
-synopsis: "Tactician: A Seamless, Interactive Tactic Learner and Prover for Coq"
+version: "8.12.dev"
```
Comment (palmskog) [inline extra-dev/packages/coq-tactician/coq-tactician.8.12.dev/opam:2]:
```suggestion
```

File: extra-dev/packages/coq-tactician/coq-tactician.8.12.dev/opam [hunk 4]
```diff
@@ -1,43 +1,51 @@
...
+authors: ["Lasse Blaauwbroek <lasse@blaauwbroek.eu>"]
 homepage: "https://coq-tactician.github.io"
-dev-repo: "git+https://github.com/coq-tactician/coq-tactician"
 bug-reports: "https://github.com/coq-tactician/coq-tactician/issues"
-maintainer: "Lasse Blaauwbroek <lasse@blaauwbroek.eu>"
-authors: "Lasse Blaauwbroek <lasse@blaauwbroek.eu"
 depends: [
   "ocaml" {>= "4.08"}
-  "dune" {>= "2.5"}
+  "dune" {>= "2.5" & build}
```
Comment (palmskog) [inline extra-dev/packages/coq-tactician/coq-tactician.8.12.dev/opam:20]:
```suggestion
  "dune" {>= "2.5"}
```

File: extra-dev/packages/coq-tactician/coq-tactician.8.13.dev/opam [hunk 5]
```diff
@@ -1,43 +1,51 @@
...
+authors: ["Lasse Blaauwbroek <lasse@blaauwbroek.eu>"]
 homepage: "https://coq-tactician.github.io"
-dev-repo: "git+https://github.com/coq-tactician/coq-tactician"
 bug-reports: "https://github.com/coq-tactician/coq-tactician/issues"
-maintainer: "Lasse Blaauwbroek <lasse@blaauwbroek.eu>"
-authors: "Lasse Blaauwbroek <lasse@blaauwbroek.eu"
 depends: [
   "ocaml" {>= "4.08"}
-  "dune" {>= "2.5"}
+  "dune" {>= "2.5" & build}
```
Comment (palmskog) [inline extra-dev/packages/coq-tactician/coq-tactician.8.13.dev/opam:20]:
```suggestion
  "dune" {>= "2.5"}
```

File: extra-dev/packages/coq-tactician/coq-tactician.dev/opam [hunk 6]
```diff
@@ -1,43 +1,51 @@
 opam-version: "2.0"
-name: "coq-tactician"
-synopsis: "Tactician: A Seamless, Interactive Tactic Learner and Prover for Coq"
+version: "dev"
```
Comment (palmskog) [inline extra-dev/packages/coq-tactician/coq-tactician.dev/opam:2]:
```suggestion
```

File: extra-dev/packages/coq-tactician/coq-tactician.8.13.dev/opam [hunk 7]
```diff
@@ -1,43 +1,51 @@
 opam-version: "2.0"
-name: "coq-tactician"
-synopsis: "Tactician: A Seamless, Interactive Tactic Learner and Prover for Coq"
+version: "8.13.dev"
```
Comment (palmskog) [inline extra-dev/packages/coq-tactician/coq-tactician.8.13.dev/opam:2]:
```suggestion
```

File: extra-dev/packages/coq-tactician/coq-tactician.dev/opam [hunk 8]
```diff
@@ -1,43 +1,51 @@
...
+authors: ["Lasse Blaauwbroek <lasse@blaauwbroek.eu>"]
 homepage: "https://coq-tactician.github.io"
-dev-repo: "git+https://github.com/coq-tactician/coq-tactician"
 bug-reports: "https://github.com/coq-tactician/coq-tactician/issues"
-maintainer: "Lasse Blaauwbroek <lasse@blaauwbroek.eu>"
-authors: "Lasse Blaauwbroek <lasse@blaauwbroek.eu"
 depends: [
   "ocaml" {>= "4.08"}
-  "dune" {>= "2.5"}
+  "dune" {>= "2.5" & build}
```
Comment (palmskog) [inline extra-dev/packages/coq-tactician/coq-tactician.dev/opam:20]:
```suggestion
  "dune" {>= "2.5"}
```

**Review by palmskog — CHANGES_REQUESTED** [review]

**Comment by LasseBlaauwbroek** [comment]
Thanks @palmskog, changes committed.

**Review by palmskog — APPROVED** [review]

**Pull request merged**